backend/inspirehep/search/api.py

```diff
@@ -100,6 +101,10 @@ def source_for_content_type(self, content_type):
         return self
 
     def execute(self, *args, **kwargs):
+        if request:
+            size = request.args.get("size", default=25, type=int)
+            if size > current_app.config.get("SEARCH_MAX_SEARCH_PAGE_SIZE", 500):
+                raise MaximumSearchPageSizeExceeded()
```

Comment:
I couldn't find a test for this. What status code are we returning? I think `400` would be appropriate.

Reply:
Ah right. I missed this. Thanks :+1: